config/servers.yml

```diff
@@ -91,6 +91,7 @@
 #        autoCommitEnabled:         true
 #        queueSize:                 1000
 #        fetchSize:                 1048576
+#        minPartitionCount:         1
```

Comment:
Actually, it is a deployment property, set via `module.x.producer.minPartitionCount` (and wildcarded equivalents - note how it is used in the tests). What's in `servers.yml` is the default, which is normally 1. We could hardcode it, too, but I guess it doesn't hurt to make that configurable.